_gtfobins/systemctl.md

```diff
@@ -0,0 +1,23 @@
+---
+functions:
+  suid:
+    - code: |
+        TF=/tmp/test.service
+        echo '[Service]
+          Type=oneshot
+          ExecStart=/bin/sh -c "id >> /tmp/output"
+          [Install]
+          WantedBy=multi-user.target' > $TF
```

Comment:
Please get rid of the 2 spaces indentation and use `>` instead of `>>`.

Reply:
I used `>` instead. For the 2 space indentation that's actually the same format you have in `nmap.md` and `awk.md` for multi-line statements. Do you still want me to change it?

Reply:
Yeah, you're right, leave it as it is, I'll take care of that.